api/transport/handler.go

```diff
@@ -117,64 +113,30 @@ type StreamHandler interface {
 	HandleStream(stream *ServerStream) error
 }
 
-// DispatchUnaryHandler calls the handler h, recovering panics and timeout errors,
-// converting them to yarpc errors. All other errors are passed trough.
+// DispatchUnaryHandler is the deprecated - use InvokeUnaryHandler instead
 func DispatchUnaryHandler(
```

Comment:
I would leave the original docs intact and add

```
  // Deprecated: Use InvokeUnaryHandler instead.
```

staticcheck recognizes that and fails check if the deprecated function is used.